### ISSUE TITLE
Customize the plotly toolbar from Dash layout

### ISSUE THREAD

**Issue by jamestwebber** [plotly/dash#102]
First off thanks for building Dash, it's super nice for making a simple dashboard of plots. One thing I've been trying to figure out is how to remove extraneous buttons from the toolbar. I don't want to remove it entirely but some options don't make sense for our use case and I'd prefer to simplify the interface.

I haven't been able to find how to do this specifically&mdash;I think I could figure out a way to hack some CSS in there and hide the buttons I don't want, but that doesn't seem like a good solution. Did I miss how to do this? If not, it would be a useful feature to add.

**Comment by chriddyp** [comment]
Thanks for the feedback! Yeah, we should make this configurable. There is a `config` argument in the [`plotly.js`](https://github.com/plotly/plotly.js) that we just need to add to the `Graph` component....
it's an easy enough fix, I'll just add it now 👍

**Comment by chriddyp** [comment]
Fix here: https://github.com/plotly/dash-core-components/pull/43

**Comment by chriddyp** [comment]
This is available in `dash-core-components` `v0.9.0` now.
Upgrade with `pip install dash-core-components==0.9.0` 

Simple example:
```
import dash
import dash_core_components as dcc
import dash_html_components as html

app = dash.Dash()

app.layout = html.Div([
    dcc.Graph(id='my-graph', figure={'data': [{'x': [1, 2, 3]}]}, config={'editable': True, 'modeBarButtonsToRemove': ['pan2d', 'lasso2d']})
])

if __name__ == '__main__':
    app.run_server(debug=True)
```

Full list of options available in `help(dcc.Graph)`

Happy Friday! 🍻
